Fixes service discovery browsing with PEP-capable
clients like Gajim, when Pubsub is served by mod_pubsub_odbc.

This issue is tracked upstream as EJAB-1115.

This changeset was introduced in revision 2784,
branch http://svn.process-one.net/ejabberd/branches/ejabberd-2.1.x

The first stable version containing the fix is 2.1.1.

Index: src/mod_pubsub/mod_pubsub.erl
===================================================================
---  a/src/mod_pubsub/mod_pubsub.erl	(revision 2783)
+++  b/src/mod_pubsub/mod_pubsub.erl	(revision 2784)
@@ -631,7 +631,7 @@
     end,
     {result, Feats ++ lists:map(fun(Feature) ->
 	?NS_PUBSUB++"#"++Feature
-    end, features(Host, []))};
+    end, features(Host, <<>>))};
 disco_local_features(Acc, _From, _To, _Node, _Lang) ->
     Acc.
 
Index: src/mod_pubsub/mod_pubsub_odbc.erl
===================================================================
---  a/src/mod_pubsub/mod_pubsub_odbc.erl	(revision 2783)
+++  b/src/mod_pubsub/mod_pubsub_odbc.erl	(revision 2784)
@@ -434,7 +434,7 @@
     end,
     {result, Feats ++ lists:map(fun(Feature) ->
 	?NS_PUBSUB++"#"++Feature
-    end, features(Host, []))};
+    end, features(Host, <<>>))};
 disco_local_features(Acc, _From, _To, _Node, _Lang) ->
     Acc.
 
